index.es.md

```diff
@@ -0,0 +1,246 @@
+-   Cuando esté preparado para saber más sobre desarrollo y diseño, probablemente le serán útiles recursos como:
+    -   [Cómo cumplir las WCAG (Referencia Rápida)](http://www.w3.org/WAI/WCAG21/quickref/)
+    -   [Tutoriales de Accesibilidad Web](https://www.w3.org/WAI/tutorials/)
+-   Para la gestión de proyectos y consideraciones organizativas, acceda a [[Planificación y Gestión de la Accesibilidad Web]](/planning-and-managing/).<br>
+    Si necesita hacer correciones rápidas, acceda a [[Estrategias para Reparaciones Provisionales]](/planning/interim-repairs/).
+
+{::nomarkdown}
+{% include box.html type="end" %}
+{:/}
+	
+## Evaluar la Accesibilidad {#evaluate}
+
+Al desarrollar o rediseñar un sitio web, evalúe la accesibilidad desde el principio y durante todo el proceso de desarrollo para identificar los problemas de accesibilidad desde el inicio, cuando es más fácil resolverlos. Pasos sencillos, como cambiar la configuración del navegador, pueden ayudarle a evaluar algunos aspectos de la accesibilidad. Una evaluación exhaustiva para determinar si un sitio web cumple con todas las pautas de accesibilidad requiere más esfuerzo.
+
+Existen herramientas que ayudan en la evaluación. Sin embargo, ninguna herramienta por sí sola puede determinar si un sitio cumple con las pautas de accesibilidad. Se requiere una evaluación humana bien experimentada para determinar si un sitio es accesible.
+
+
+{::nomarkdown}
+{% include box.html type="start" h="3" title="Más información sobre cómo Evaluar Accesibilidad" class="simple aside" %}
+{:/}
+
+-   Los recursos para ayudar en la evaluación de accesibilidad se describen en [[Evaluar la Accesibilidad de Sitios Web]](/test-evaluate/).
+
+{::nomarkdown}
+{% include box.html type="end" %}
+{:/}
+
+{% include excol.html type="start" id="examples" %}
+
+## Ejemplos
+
+{% include excol.html type="middle" %}
+
+### Alternativas textuales para imágenes
+
+![image of logo; HTML markup img alt='Iniciativa de Accesibilidad Web'](https://www.w3.org/WAI/intro/alt-logo.png){:.right}
+
+Las imágenes deben incluir una *[alternativa textual equivalente](http://www.w3.org/TR/UNDERSTANDING-WCAG20/text-equiv.html)* (texto alternativo) en el marcado/código.
+
+Si no se facilita texto alternativo para las imágenes, su información es inaccesible, por ejemplo para personas que no pueden ver y utilizan un lector de pantalla que lee la información de la página, incluyendo el texto alternativo de la imagen.
+
+Cuando el equivalente textual es facilitado, la información está disponible para personas ciegas, además de para personas que desactivan las imágenes (por ejemplo, en áreas con ancho de banda limitado o de alto coste). También está disponible para tecnologías que no pueden ver imágenes, como por ejemplo los motores de búsqueda.
+
+### Entrada con teclado
+
+![mouse crossed out](https://www.w3.org/WAI/intro/no-mouse.png){:.left width="67" height="45"}
```

Comment:
"mouse crossed out"

needs translation

Reply:
"Mouse crossed out" could be translated as "Ratón tachado con una cruz"